includes/managers/class-fs-plan-manager.php

```diff
@@ -106,32 +106,59 @@ function has_free_plan( $plans ) {
 			return false;
 		}
 
-		/**
-		 * Find all plans that have trial.
-		 *
-		 * @author Vova Feldman (@svovaf)
-		 * @since  1.0.9
-		 *
-		 * @param FS_Plugin_Plan[] $plans
-		 *
-		 * @return FS_Plugin_Plan[]
-		 */
-		function get_trial_plans( $plans ) {
-			$trial_plans = array();
-
-			if ( is_array( $plans ) && 0 < count( $plans ) ) {
-				/**
-				 * @var FS_Plugin_Plan[] $plans
-				 */
-				for ( $i = 0, $len = count( $plans ); $i < $len; $i ++ ) {
-					if ( $plans[ $i ]->has_trial() ) {
-						$trial_plans[] = $plans[ $i ];
-					}
-				}
-			}
-
-			return $trial_plans;
-		}
+        /**
+         * Find all plans that have trial.
+         * Since 2.6.2 call get_filtered_plan
+         *
+         * @author Vova Feldman (@svovaf)
+         * @since  1.0.9
+         *
+         * @param FS_Plugin_Plan[] $plans
+         *
+         * @return FS_Plugin_Plan[]
+         */
+        function get_trial_plans( $plans ) {
+            return $this->get_filtered_plans($plans, true);
```

Comment:
This repository kind of follows a lose WP Coding guideline. So extra spacing is expected

```php
return $this->get_filtered_plans( $plans, true );
```

Please see to all the code and kindly fix the formatting.